pkg/custompluginmonitor/custom_plugin_monitor.go

```diff
@@ -143,6 +150,17 @@ func (c *customPluginMonitor) generateStatus(result cpmtypes.Result) *types.Stat
 	}
 }
 
+func toConditionStatus(s cpmtypes.Status) types.ConditionStatus {
+	switch s {
+	case cpmtypes.OK:
+		return types.True
```

Comment:
when the check script returns `OK`, the condition status should return `types.False` which indicates that the condition is `False`.

Reply:
True.

Reply:
Done